app/serializers/profile_serializer.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class ProfileSerializer < ActiveModel::Serializer
+  attributes(
+    :id,
+    :first_names,
+    :last_name,
+    :date_of_birth,
+    :assessment_answers,
+    :identifiers,
+    :gender_additional_information,
+  )
+
+  # although we are serialising a 'profile', to the outside world this is a person.
+  type 'people'
+
+  has_one :ethnicity, serializer: EthnicitySerializer
+  has_one :gender, serializer: GenderSerializer
+
+  def id
+    object.person.id
```

Comment:
@starswan should this definitely be the `person.id` and not the `profile.id` ? There is a danger that different profiles for the same person could be returned - each with the same apparent id. That might confuse api consumers?

Reply:
Yes - there's no profile endpoint. The profile returned by the person endpoint is always the 'latest_profile' (yes I'm not sure whether this works either)